src/components/data/PowerBI/Report/components/view/PowerBIReportView.tsx

```diff
@@ -47,10 +45,9 @@ export const PowerBIReportView: FC<PowerBIComponentProps> = ({ config }: PowerBI
         [event$]
     );
 
-    const getEmbeddedComponent = useCallback(
-        (value) => (component?.current ? (component.current = value) : null),
-        [component]
-    );
+    const getEmbeddedComponent = (value) => {
+        component.current = value;
```

Comment:
Seems to be some type error here? `getEmbeddedComponent` should receive `value: Embed` but then cannot assign `component.current` to valuje because it's of type `PowerBIEmbedComponent`. Still works though..